src/scalper.ts

```diff
@@ -1064,15 +1047,14 @@ Spread % ${((whaleAskPrice - whaleBidPrice) / fairValue) * 100}`);
     } catch (err) {
       console.log(this.symbol, 'Gamma Order', err, err.stack);
     }
-    console.log(
-      this.symbol,
-      'Gamma Spread %',
-      ((priceAsk - priceBid) / fairValue) * 100,
-      'Liquidity $',
-      (amountGamma * 2 + backBidQty + backAskQty) * fairValue,
-    );
+    console.log(`${this.symbol} Gamma Spread % ${((priceAsk - priceBid) / fairValue) * 100} \
+Liquidity $ ${(amountGamma * 2 + backBidQty + backAskQty) * fairValue}`);
+
+    // At the base level only, wait for the scalp to fill.
     if (gammaScalpCount === 1) {
       await waitForFill((_) => gammaScalpCount === gammaCycles, scalperWindowSec);
+
+      // TODO: Remove this logging unless necessary. It is not showing new actionable info.
```

Comment:
We should try to record scalps per run potentially but so long as we have good pnl accounting which we dont yet can remove. Useful to have a log when the gamma run has completed still

Reply:
Logging should not be for accounting. We can have separate accounting scripts. Logs are just there for debugging and streamlined to critical messages for fixing an outage.
Can add something in the give up of wait for fill if needed